src/App.jsx

```diff
@@ -18,6 +19,7 @@ function App() {
     <Router>
       <Header />
       <Switch>
+        <Route path="/scholarships/new" component={ScholarshipForm} />
```

Comment:
Can we add a button for this in the header? Otherwise we have to remember the url path and it's not discoverable